backend/prisma/migrations/20200408071119-init/schema.prisma

```diff
@@ -0,0 +1,118 @@
+datasource db {
```

Comment:
Just checking/curious, is this file and the `readme.md` file in `backend/prisma/migrations/20200408071119-init/README.md` needed or can we delete them?

Reply:
Yep they are needed 🙂

Reply:
@robin-macpherson ahh ok, just wanted to double check cause the column values were spelled differently, so I thought we might need to generate a new migration